manifest.json

```diff
@@ -604,6 +604,14 @@
       "type": "library",
       "version": "22.0.1"
     },
+    {
+      "id": "keyhud",
+      "description": "Simple *key HUD* for lite-xl"
```

Comment:
Missing comma.
Same thing in the manifest in your repo.

Also, might be better to use `Lite XL` instead of `lite-xl`.

Reply:
Thank you very much, I did not pay attention when I created the manifest on my repo and then copy pasted it... sorry

I changed `lite-xl` to `Lite XL` :)